### ISSUE TITLE
Rake install does not compile any more.

### ISSUE THREAD

**Issue by loloof64** [ruboto/ruboto#830]
In my ruboto project, I managed to include JRuby jar and launch application on the emulator without Ruboto core. Then I tried to include external gem by adding it into Gemfile.apk, but now the project does not compile any more.

This is the Gemfile.apk

    source "http://rubygems.org"
    gem 'pgn'

My ruboto

    $ ruboto --version
    1.5.0

My OS is Ubuntu 16.10 64 bits, and I am using rbenv as the ruby installer.

The error file is the standard error from the command `rake install start`

[ruboto_error.txt](https://github.com/ruboto/ruboto/files/679560/ruboto_error.txt)

**Comment by donv** [comment]
Hi @loloof64 !

Did you figure this out?

**Comment by loloof64** [comment]
Hi @donv,

thanks for your interest, I must admit that very soon since this problem occured, I've given up. I did not manage to solve it.

**Comment by donv** [comment]
Well @loloof64 , if you would like to give it another shot, please let me know.  Closing for now.

**Comment by loloof64** [comment]
It's ok to give a new try

**Comment by loloof64** [comment]
Hi again @donv !

I tested `rake install start` again on the same project (setups for Ruboto and Bundler are fine), but I get another error. This one come from Ant, Rake issue seems to have been resolved. But I still don't know why I can't launch my project on emulator.

[ruboto_error.txt](https://github.com/ruboto/ruboto/files/847632/ruboto_error.txt)

I shared [my project on my github account](https://github.com/loloof64/ChessExercisesToolRuboto)
